src/generate.d.ts

```diff
@@ -1,12 +1,52 @@
-interface Options {
-  length?: number
-  numbers?: boolean
-  symbols?: boolean
-  lowercase?: boolean
-  uppercase?: boolean
-  excludeSimilarCharacters?: boolean
-  exclude?: string
-  strict?: boolean
+export interface GenerateOptions {
```

Comment:
does renaming `Options` to `GenerateOptions` result in backwards compatibility issues?

Reply:
I don't think it does because the `Options` interface wasn't exported before.
And the main reason behind the rename is the export. I thought now that people can import the interface, it's better to have a more specific name for it.